src/main/com/example009/radek/v3/FileService.java

```diff
@@ -0,0 +1,93 @@
+package main.com.example009.radek.v3;
+
+import java.io.*;
+import java.util.List;
+import java.util.Scanner;
+
+public class FileService {
+
+    final private String path = "src/main/com/example009/radek/v3/calculator-results/";
+
+    void storeResultsIntoFile(List<Double> inputList) {
+
+        String name = getUserInput("Please insert file name: ", false);
+        saveResultsIntoFile(name, inputList);
+    }
+
+    void loadResultsFromFile() {
+
+        try {
+            String name = getUserInput("Please insert file name: ", true);
+
+            BufferedReader reader = new BufferedReader(new FileReader(path + name));
+            reader.lines().forEach(System.out::println);
+            reader.close();
+        } catch (FileNotFoundException e) {
+            System.out.println("Connecting to file reader wasn't successful");
```

Comment:
vyhazujes exception File not found, ale podle vypisu selhalo nejake pripojeni k souboru ... 
pro exception pouzivej nove System.err

e printStackTrace se nepouziva, ale pro testovaci ucely to je ok,
lepri by bylo neco jako throw new Exception / FileNotFoundExcpetion ... a v konstruktorui prislusny text

Reply:
Upraveno